package.json

```diff
@@ -87,6 +88,7 @@
     "lab": "^14.2.0",
     "napa": "^2.3.0",
     "pg-copy-streams": "^1.2.0",
+    "postgrator": "^2.10.3",
```

Comment:
was this added on purpose?

Reply:
Oh not sure how that got there, let me remove it!